modules/gsuite_enabled/main.tf

```diff
@@ -38,7 +38,17 @@ module "gsuite_group" {
   domain = "${var.domain}"
   name   = "${local.group_name}"
   org_id = "${var.org_id}"
-  create_group = "${var.create_group}"
+}
+
+/******************************************
+  Gsuite Group Configuration
+ *****************************************/
+resource "gsuite_group" "group" {
+  count = "${var.create_group ? 1 : 0}"
+
+  description = "${var.name} project group"
+  email       = "${module.gsuite_group.email}"
+  name        = "${var.name}"
```

Comment:
Should this be `local.group_name`? `var.name` is the name of the project itself, right?

Reply:
Done.